database/factories/CategoryFactory.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace Database\Factories;
+
+use Illuminate\Database\Eloquent\Factories\Factory;
+use App\Models\Category;
+/**
+ * @extends \Illuminate\Database\Eloquent\Factories\Factory<\App\Models\Category>
+ */
+class CategoryFactory extends Factory
+{
+    protected $model = Category::class;
+    protected static $names_map =[
+        'Demonstratie',
+        'Petitie',
+    ];
+
+    public function definition(): array
+    {
+
+        # This number is the same in the count parameter in the seeder file
+        $tot_elements = $this->count;
+
+        $name = $this->faker->unique()->randomElement(static::$names_map);
```

Comment:
Since this variable `$name` is used only once, it might be good to just call `$this->faker->unique()->randomElement(static::$names_map)` directly in line 37

Reply:
I'm using the $name to generate the slug field as well, in the line 38.